### DISCUSSION TITLE
What is the difference between "reload" and "forced reload" in Chia GUI ?

### DISCUSSION THREAD

**Discussion by RollingRoland** [Chia-Network/chia-blockchain#3162] [General]
I have frequently issues with my Chia GUI going blank or hanging itself. Under menu tabs "view" there are the options "reload" and "forced reload". Once when my GUI went blank "reload" brought it back. But I have another issue now where the GUI is now hanging as well under "reload". I have 3 plots being created in the background. So I am reluctant in trying "forced reload". Because I am afraid it would kill the unfinished plots.

Does anyone know what the exact difference is between "reload" and "forced reload" ?

**Comment by StrikeNP** [comment]
The Chia gui is basically a web browser. Here's a relevant excerpt from a blog:

> What is a Hard Refresh? How to do a Hard Refresh in Any Browser.
> A hard refresh clears your browser cache for a specific page, which forces it to load the most recent version of that page. This could include new scripts, styles or features.
> 
> Important: Clicking the “Refresh” arrow on your browser address bar is not a hard refresh. Here’s how you can clear your cache through a hard refresh on any browser:

https://www.documate.org/automation/what-is-a-hard-refresh-how-to-do-a-hard-refresh-in-any-browser/

In practice, no the refresh will not stop your plots in my experience. However, there is currently a bug where refreshing while plotting will get you stuck at a "connecting to wallet" screen, which you will have to close the app (lose your plots) and start again. I've had to do this many times :-(
I have also noticed that if the gui crashes (but you don't close it, it just disappears) the currently plotting plots will finish, but any queued plots will not.

**Comment by sargonas** [comment]
Tagging this issue for migration to a discussion. Apologies in advance, but you will be best served by moving this over there and out of the issue tracker, where the community can better assist you. (Ideally the issue tracker should be used to focus on bugs and development problems that need to be addressed, with general user support handled in Discussions and Keybase)

^MigrateToDiscussion
